bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/DefaultPerChannelBookieClientPool.java

```diff
@@ -48,6 +48,7 @@
     final BookieSocketAddress address;
 
     final PerChannelBookieClient[] clients;
+    final PerChannelBookieClient[] clientsV3;
```

Comment:
It is better to call it `clientsV3Enforced`. otherwise it becomes a bit confused, because `clients` can be v3 as well.

Reply:
good idea